src/main/xerus/monstercat/api/Cache.kt

```diff
@@ -38,10 +38,15 @@ object Cache: Refresher() {
 		return releases
 	}
 	
-	/** Gets all tracks by flatMapping all the tracks of all Releases */
+	/** Gets all tracks by flatMapping all the tracks of all Releases. Two tracks with the same id will only yield one (hashSet) */
 	suspend fun getTracks(): Collection<Track> =
 		getReleases().flatMap { it.tracks }.toHashSet()
 	
+	/** Gets all tracks by flatMapping all the tracks of all Releases. Will return all tracks, regardless of id duplicates.
+	 * This is very useful when you need all tracks for a release.*/
```

Comment:
```suggestion
	 * This is very useful when you need a separate Track for each Release it is part of. */
```

Reply:
All clear for me.

Reply:
cd4aa813bc62828d7cc18144b4bbb45f772e8223